### PR TITLE
fixed minor spelling mistakes

### PR DESCRIPTION
**Pull request by Shail-Shouryya**

### CODE REVIEW

File: presidio-analyzer/analyzer/entity_recognizer.py [hunk 1]
```diff
@@ -276,12 +276,12 @@ def find_index_of_match_token(word, start, tokens, tokens_indices):
                              "is expected to be found")
         return i
 
-    def __extract_surrounding_words(self, nlp_artifacts, word, start):
-        """ Extracts words surronding another given word.
+    def __extract_context(self, nlp_artifacts, word, start):
```
Comment (omri374) [inline presidio-analyzer/analyzer/entity_recognizer.py]: function name has changed

Comment (Shail-Shouryya) [reply presidio-analyzer/analyzer/entity_recognizer.py]: missed that, thanks for catching it!

Comment (omri374) [reply presidio-analyzer/analyzer/entity_recognizer.py]: Please fix :)

Comment (Shail-Shouryya) [reply presidio-analyzer/analyzer/entity_recognizer.py]: Sorry for the slow update, I'm still learning about Git and thought the PR review meant it was fixed. Fixed and committed now :)

Comment (omri374) [reply presidio-analyzer/analyzer/entity_recognizer.py]: Awesome. Thanks so much!

**Pull request merged**